constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
It seems to me that the constraints files checked into master should always be empty.  Ensure this by checking in an empty file, committing, and then adding a line for `constraints.txt` to the `.gitignore` file, and committing again.

If you think the constraints files should not always be empty in master, I would like to hear why.  It seems like then we would have a master branch that has dependencies on non-master branches of ga4gh code, which doesn't seem right.